style.css

```diff
@@ -0,0 +1,74 @@
+/* stylelint-disable -- giving errors in font style */
```

Comment:
[OPTIONAL] Please try not to disable linters, if you are having errors, try to add font link to CSS instead of HTML head.👌